<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>PYTHON API TEST RESULT</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">PYTHON API TEST RESULT</h1>
<p class='attribute'><strong>测试人员 : </strong> mai</p>
<p class='attribute'><strong>开始时间 : </strong> 2019-05-21 13:50:06</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:04.176108</p>
<p class='attribute'><strong>测试结果 : </strong> 共 37，通过 23，失败 4，错误 10，通过率= 62.16%</p>

<p class='description'>webserver</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 62.16% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 4 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 23 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 37 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>test_bindBankCard.TestbindBankCard</td>
    <td class="text-center">4</td>
    <td class="text-center">3</td>
    <td class="text-center">0</td>
    <td class="text-center">1</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',4)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='pt1_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testbindBankCard_1</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_1'>通过</button>
    <div id='div_pt1_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_1'>通过</button>
    <div id='div_pt1_1' class="collapse in" align="left">
    <pre>
    
pt1_1: 替換后的sql是: select Fverify_code from sms_db_94.t_mvcode_info_5 where Fmobile_no =18201874594


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testbindBankCard_2</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testbindBankCard_3</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft1_4' class='none'>
    <td class='errorCase'><div class='testcase'>testbindBankCard_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse in" align="left">
    <pre>
    
ft1_4: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_bindBankCard.py", line 42, in testbindBankCard
    actual = dohttp(case.url, case.data, case.method)
  File ".\API2\common\do_http.py", line 22, in dohttp
    data = eval(data)
  File "&lt;string&gt;", line 1
    {'uid':'100011243','pay_pwd':'12345678','mobile':'18201874594','cre_id':'440002199807292581','user_name':'海明威','cardid':'18218813163566461145','bank_type':'1001','bank_name':'深圳南山區招商銀行分局','bank_area':'',bank_city'':''}
                                                                                                                                                                                                                         ^
SyntaxError: invalid syntax


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>test_sendMCode.TestsendMCode</td>
    <td class="text-center">4</td>
    <td class="text-center">3</td>
    <td class="text-center">0</td>
    <td class="text-center">1</td>
    <td class="text-center"><a href="javascript:showClassDetail('c2',4)" class="detail" id='c2'>详细</a></td>
</tr>

<tr id='ft2_1' class='none'>
    <td class='errorCase'><div class='testcase'>testsendMCode_1</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse in" align="left">
    <pre>
    
ft2_1: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_sendMCode.py", line 36, in testsendMCode
    before_count = self.domysql.get_fetchone(eval(case.sql)['sql_count'])[0]
  File ".\API2\common\do_sql.py", line 14, in get_fetchone
    self.cursor.execute(sql)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='pt2_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testsendMCode_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt2_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt2_2'>通过</button>
    <div id='div_pt2_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt2_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt2_2'>通过</button>
    <div id='div_pt2_2' class="collapse in" align="left">
    <pre>
    
pt2_2: &lt;suds.sax.document.Document object at 0x00000218CDD44470&gt;
手机号码错误


    </pre>
    </div>
    </td>
</tr>

<tr id='pt2_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testsendMCode_3</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt2_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt2_3'>通过</button>
    <div id='div_pt2_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt2_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt2_3'>通过</button>
    <div id='div_pt2_3' class="collapse in" align="left">
    <pre>
    
pt2_3: &lt;suds.sax.document.Document object at 0x00000218CDD255C0&gt;
手机号码错误


    </pre>
    </div>
    </td>
</tr>

<tr id='pt2_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testsendMCode_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt2_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt2_4'>通过</button>
    <div id='div_pt2_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt2_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt2_4'>通过</button>
    <div id='div_pt2_4' class="collapse in" align="left">
    <pre>
    
pt2_4: &lt;suds.sax.document.Document object at 0x00000218CDF9C828&gt;
手机号码错误


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>unittest.suite._ErrorHolder</td>
    <td class="text-center">3</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">3</td>
    <td class="text-center"><a href="javascript:showClassDetail('c3',3)" class="detail" id='c3'>详细</a></td>
</tr>

<tr id='ft3_1' class='none'>
    <td class='errorCase'><div class='testcase'>TestsendMCode)</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_1'>错误</button>
    <div id='div_ft3_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_1'>错误</button>
    <div id='div_ft3_1' class="collapse in" align="left">
    <pre>
    
ft3_1: &lt;suds.sax.document.Document object at 0x00000218CDF9C828&gt;
手机号码错误
Traceback (most recent call last):
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_sendMCode.py", line 65, in tearDownClass
    cls.domysql.close()
  File ".\API2\common\do_sql.py", line 25, in close
    self.mysql.close()
  File "C:\Python37\lib\site-packages\pymysql\connections.py", line 354, in close
    raise err.Error("Already closed")
pymysql.err.Error: Already closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_2' class='none'>
    <td class='errorCase'><div class='testcase'>TestuserRegister)</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse in" align="left">
    <pre>
    
ft3_2: Traceback (most recent call last):
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_userRegister.py", line 78, in tearDownClass
    cls.domysql.close()
  File ".\API2\common\do_sql.py", line 25, in close
    self.mysql.close()
  File "C:\Python37\lib\site-packages\pymysql\connections.py", line 354, in close
    raise err.Error("Already closed")
pymysql.err.Error: Already closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_3' class='none'>
    <td class='errorCase'><div class='testcase'>TestverifyUserAuth)</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_3'>错误</button>
    <div id='div_ft3_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_3'>错误</button>
    <div id='div_ft3_3' class="collapse in" align="left">
    <pre>
    
ft3_3: Traceback (most recent call last):
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 77, in tearDownClass
    cls.domysql.close()
  File ".\API2\common\do_sql.py", line 25, in close
    self.mysql.close()
  File "C:\Python37\lib\site-packages\pymysql\connections.py", line 354, in close
    raise err.Error("Already closed")
pymysql.err.Error: Already closed


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>test_userRegister.TestuserRegister</td>
    <td class="text-center">16</td>
    <td class="text-center">13</td>
    <td class="text-center">1</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c4',16)" class="detail" id='c4'>详细</a></td>
</tr>

<tr id='ft4_1' class='none'>
    <td class='errorCase'><div class='testcase'>testuserRegister_01</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_1'>错误</button>
    <div id='div_ft4_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_1'>错误</button>
    <div id='div_ft4_1' class="collapse in" align="left">
    <pre>
    
ft4_1: 替換后的sql是: select Fverify_code from sms_db_48.t_mvcode_info_5 where Fmobile_no =18263713548
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_userRegister.py", line 72, in testuserRegister
    raise e
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_userRegister.py", line 68, in testuserRegister
    self.mobile.make_code(case.sql)
  File ".\API2\common\mobliephone.py", line 20, in make_code
    mobile_code = DoMysql().get_fetchone(sql)[0]
  File ".\API2\common\do_sql.py", line 14, in get_fetchone
    self.cursor.execute(sql)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_2' class='none'>
    <td class='errorCase'><div class='testcase'>testuserRegister_02</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_2'>错误</button>
    <div id='div_ft4_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_2'>错误</button>
    <div id='div_ft4_2' class="collapse in" align="left">
    <pre>
    
ft4_2: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_userRegister.py", line 44, in testuserRegister
    before_count = self.domysql.get_fetchone(case.sql)[0]
  File ".\API2\common\do_sql.py", line 14, in get_fetchone
    self.cursor.execute(sql)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_3' class='none'>
    <td class='failCase'><div class='testcase'>testuserRegister_03</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_3'>失败</button>
    <div id='div_ft4_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_3'>失败</button>
    <div id='div_ft4_3' class="collapse in" align="left">
    <pre>
    
ft4_3: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_userRegister.py", line 58, in testuserRegister
    raise e
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_userRegister.py", line 53, in testuserRegister
    self.unit.assertEqual(case.expect, actual)
AssertionError: '该用户已存在' != '验证码错误'
- 该用户已存在
+ 验证码错误



    </pre>
    </div>
    </td>
</tr>

<tr id='pt4_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_04</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_5' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_05</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_6' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_06</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_7' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_07</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_8' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_08</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_9' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_09</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_10' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_10</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_11' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_11</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_12' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_12</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_13' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_13</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_14' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_14</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_15' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_15</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_16' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testuserRegister_16</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr class='errorClass warning'>
    <td>test_verifiedUserAuth.TestverifyUserAuth</td>
    <td class="text-center">10</td>
    <td class="text-center">4</td>
    <td class="text-center">3</td>
    <td class="text-center">3</td>
    <td class="text-center"><a href="javascript:showClassDetail('c5',10)" class="detail" id='c5'>详细</a></td>
</tr>

<tr id='ft5_1' class='none'>
    <td class='errorCase'><div class='testcase'>testverifyUserAuth_01</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse in" align="left">
    <pre>
    
ft5_1: 替換后的sql是: select Fverify_code from sms_db_82.t_mvcode_info_3 where Fmobile_no =18216362382
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 71, in testverifyUserAuth
    raise e
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 67, in testverifyUserAuth
    self.mobile.make_code(case.sql)
  File ".\API2\common\mobliephone.py", line 20, in make_code
    mobile_code = DoMysql().get_fetchone(sql)[0]
  File ".\API2\common\do_sql.py", line 14, in get_fetchone
    self.cursor.execute(sql)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_2' class='none'>
    <td class='failCase'><div class='testcase'>testverifyUserAuth_02</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_2'>失败</button>
    <div id='div_ft5_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_2'>失败</button>
    <div id='div_ft5_2' class="collapse in" align="left">
    <pre>
    
ft5_2: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 58, in testverifyUserAuth
    raise e
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 52, in testverifyUserAuth
    self.unit.assertEqual(case.expect,actual)
AssertionError: 'ok' != '验证码错误'
- ok
+ 验证码错误



    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_3' class='none'>
    <td class='failCase'><div class='testcase'>testverifyUserAuth_03</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_3'>失败</button>
    <div id='div_ft5_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_3'>失败</button>
    <div id='div_ft5_3' class="collapse in" align="left">
    <pre>
    
ft5_3: &lt;suds.sax.document.Document object at 0x00000218CE416C18&gt;
Unmarshalling Error: For input string: "" 
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 58, in testverifyUserAuth
    raise e
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 52, in testverifyUserAuth
    self.unit.assertEqual(case.expect,actual)
AssertionError: '用户Uid错误' != 'Unmarshalling Error: For input string: "" '
- 用户Uid错误
+ Unmarshalling Error: For input string: "" 



    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_4' class='none'>
    <td class='failCase'><div class='testcase'>testverifyUserAuth_04</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_4'>失败</button>
    <div id='div_ft5_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_4'>失败</button>
    <div id='div_ft5_4' class="collapse in" align="left">
    <pre>
    
ft5_4: &lt;suds.sax.document.Document object at 0x00000218CDCB4BA8&gt;
Unmarshalling Error: For input string: "nihao" 
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 58, in testverifyUserAuth
    raise e
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 52, in testverifyUserAuth
    self.unit.assertEqual(case.expect,actual)
AssertionError: '用户Uid错误' != 'Unmarshalling Error: For input string: "nihao" '
- 用户Uid错误
+ Unmarshalling Error: For input string: "nihao" 



    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_5' class='none'>
    <td class='errorCase'><div class='testcase'>testverifyUserAuth_05</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_5'>错误</button>
    <div id='div_ft5_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_5'>错误</button>
    <div id='div_ft5_5' class="collapse in" align="left">
    <pre>
    
ft5_5: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 43, in testverifyUserAuth
    fuid = self.domysql.get_fetchone(case.sql)[0]
  File ".\API2\common\do_sql.py", line 14, in get_fetchone
    self.cursor.execute(sql)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='pt5_6' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testverifyUserAuth_06</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt5_7' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testverifyUserAuth_07</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt5_8' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testverifyUserAuth_08</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt5_9' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>testverifyUserAuth_09</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft5_10' class='none'>
    <td class='errorCase'><div class='testcase'>testverifyUserAuth_10</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_10'>错误</button>
    <div id='div_ft5_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_10'>错误</button>
    <div id='div_ft5_10' class="collapse in" align="left">
    <pre>
    
ft5_10: Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\PycharmProjects\webserver_test2\API2\testcases\test_verifiedUserAuth.py", line 39, in testverifyUserAuth
    before_count = self.domysql.get_fetchone(case.sql)[0]
  File ".\API2\common\do_sql.py", line 14, in get_fetchone
    self.cursor.execute(sql)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Python37\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>37</td>
    <td>23</td>
    <td>4</td>
    <td>10</td>
    <td>通过率：62.16%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
